.github/workflows/doc.yml

```diff
@@ -26,10 +25,12 @@ jobs:
       - name: Generate
         run: |
           python -m pip install .
-          python -m pip install ".[doc]"
-          make docs
-          mv ding/docs/build/html public
-          rm -rf ding/docs/build
+          git clone https://github.com/opendilab/DI-engine-docs.git
```

Comment:
Consider to make a sync with the version of doc repo?

Reply:
just restrict to `main` branch now?

Reply:
Well if you need to edit code and doc, especially when you are using branch, this way may cause problem because the verison of doc not the same.